zebra/zebra_vxlan.c

```diff
@@ -7836,6 +7836,52 @@ void zebra_vxlan_remote_macip_add(ZAPI_HANDLER_ARGS)
 	return;
 }
 
+/*
+ * Handle remote vtep delete by kernel; re-add the vtep if we have it
+ */
+int zebra_vxlan_check_readd_vtep(struct interface *ifp,
```

Comment:
just a function's name is confusing :) `read-d` vs. `re-add`. of course, a comment helps.

Reply:
there is a comment right above it? it says re-add?

Reply:
we already have a pattern of using `_readd_` in the code base.  I would prefer to keep it.